README.md

```diff
@@ -1 +1,41 @@
-# first-capstone
\ No newline at end of file
+# capstone project
+
+> Setting up a portfolio desktop page.
+## Built With
+
+- HTML
+- CSS
+
+## contributor
+
+## resources
+Google
+
+
+## Author
+👤 **Richard Friday**
+
+- GitHub: [@Richez003](https://github.com/Richez003)
+- Twitter: [@richardfriday14](https://twitter.com/richardfriday14)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/richard-friday-54980718a)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Please deploy your page and add a live demo in the README

Reply:
done